common/components/BalanceSidebar/AccountInfo.jsx

```diff
@@ -0,0 +1,104 @@
+// @flow
+import './AccountInfo.scss';
+import React from 'react';
+import translate from 'translations';
+import { Identicon } from 'components/ui';
+import { formatNumber } from 'utils/formatters';
+import type Big from 'bignumber.js';
+import type { BaseWallet } from 'libs/wallet';
+import type { NetworkConfig } from 'config/data';
+
+type Props = {
+  balance: Big,
```

Comment:
I have balances / anything related to units typed out in `use-unit-types`. I'll probably go back through and convert this (and anything else this PR introduces) before PRing `use-unit-types`.